### PR TITLE
Add plots of validation and training loss

### PR DESCRIPTION
**Pull request by lo-b**
Refactor some code and add plots of losses.

### CODE REVIEW

**Comment by lo-b** [comment]
https://github.com/lo-b/basic_asr/pull/6#commitcomment-73340471 now correctly shows that the model's performance isn't great (because we shrank the training dataset to speed up workflow/checks completion times)
